### ISSUE TITLE
Add a Nekogram X app folder

### ISSUE THREAD

**Issue by flexagoon** [deckerst/aves#191]
**Is your feature request related to a problem? Please describe.**
The media folder of Nekogram X (a telegram client) is not recognized as an app folder because it's called NekoX

**Describe the solution you'd like**
In #179 you said that you already have two hard-coded app album names for KakaoTalk and Imaging Edge. I'd also like to have the `NekoX` folders recognized as belonging to the Nekogram X app

**Comment by deckerst** [comment]
I'll add it to this ID: `nekox.messenger`.

**Comment by deckerst** [comment]
@flexagoon I just noticed you wrote "NekoX folders" plural. Do you mean that there are multiple subfolders inside a parent "NekoX" folder?

**Comment by flexagoon** [comment]
@deckerst no, I just meant that there are the `Pictures/NekoX` and the `Movies/NekoX` folders
